content/blog/2020-09-09-september-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,191 @@
+---
+title: September '20 Heartbeat
+date: 2020-09-09
+description: |
+  This month, catch us on the Software Engineering Daily Podcast, 
+  check out our favorite DVC and CML tutorials and projects, and 
+  celebrate 1000 YouTube subscribers!
+descriptionLong: |
+  This month, catch us on the Software Engineering Daily Podcast, 
+  check out our favorite DVC and CML tutorials and projects, and 
+  celebrate 1000 YouTube subscribers!
```

Comment:
I guess we can make YouTube a link to get even more subscribers - DescriptionLong should support that. Also we can make DVC and CML links?